File: BlogWebApp/Blog.Web/Contracts/V1/Responses/UsersResponses/RoleResponse.cs
namespace Blog.Web.Contracts.V1.Responses.UsersResponses
{
    /// <summary>
    /// Role response.
    /// </summary>
    public class RoleResponse
    {
        /// <summary>
        /// Gets or sets the identifier.
        /// </summary>
        /// <value>
        /// The identifier.
        /// </value>
        public string Id { get; set; }

        /// <summary>
        /// Gets or sets the role.
        /// </summary>
        /// <value>
        /// The role.
        /// </value>
        public string Role { get; set; }
    }
}

File: Shared/Blog.FSharp.Contracts/V1/Responses/UsersResponses/RoleResponse.fs
namespace Blog.FSharp.Contracts.V1.Responses.UsersResponses

/// <summary>
/// Role response.
/// </summary>
type RoleResponse() =
    /// <summary>
    /// Gets or sets the identifier.
    /// </summary>
    member val Id: string = "" with get, set

    /// <summary>
    /// Gets or sets the name.
    /// </summary>
    member val Name: string = "" with get, set

